src/lib/utils.ts

```diff
@@ -7,7 +7,7 @@ export const responseWrapper = (
   endpoint: string,
   statusCode: number,
   method: string | undefined,
-  json: { [key: string]: any } = {}
```

Comment:
Nitpick since this is the way we are currently using it, but we may want to leave this more open than just `string`. It might be good to have the ability to log the body of a form input for example, in which case we'd want to support passing an object here.

Reply:
good call, was thinking the same as well. I'll play around with trying some types, but might just revert this back to `any` and just add lint disable to it.